lib/autoload/kuroko2/workflow/engine.rb

```diff
@@ -164,6 +176,35 @@ def notify_long_elapsed_time_if_needed(token)
           Kuroko2.logger.info(message)
         end
       end
+
+      def auto_retry(node, token)
+        set_sleep_context_before_retrying(node, token)
+
+        token.context['RETRY'][node.path]['retried_count'] += 1
+
+        message = "(token #{token.uuid}) Retry current node: '#{node.type}: #{node.option}'"
+        token.job_instance.logs.info(message)
+        Kuroko2.logger.info(message)
+
+        message = "(token #{token.uuid}) The number of retries: " << 
+          "#{token.context['RETRY'][node.path]['retried_count']} / #{token.context['RETRY'][node.path]['count']}"
+        token.job_instance.logs.info(message)
+        Kuroko2.logger.info(message)
+
+        message = "(token #{token.uuid}) Sleep for #{token.context['RETRY'][node.path]['sleep_time']} seconds"
```

Comment:
I want to move `set_sleep_context_before_retrying(node, token)` [L181](https://github.com/cookpad/kuroko2/pull/83/files#diff-f7b019ffd787516ae6ba13584edcd76dR181) to before this message.

Reply:
I have modified it in 2441c90